### PR TITLE
[[ docs ]] Add tag to send command

### PR DESCRIPTION
**Pull request by asayd**

### CODE REVIEW

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 05c1b9c

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 05c1b9cc4a657923c7208d8bbf41d34c707f7a3d

**Comment by montegoulding** [comment]
@livecode-vulcan review not ok 05c1b9c

File: docs/dictionary/command/send.lcdoc [hunk 1]
```diff
@@ -103,3 +103,4 @@ parameter (glossary), statement (glossary), handler (glossary),
 word (keyword), mouseUp (message), callbacks (property),
 backgroundBehavior (property)
 
+Tags: messages
```
Comment (montegoulding) [inline docs/dictionary/command/send.lcdoc:106]: @asayd could you please add a trailing newline to this file

**Comment by livecode-vulcan** [comment]
💔 review by @montegoulding not ok 05c1b9cc4a657923c7208d8bbf41d34c707f7a3d

**Comment by asayd** [comment]
@montegoulding It looks like the newline is already there.

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 05c1b9c

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 05c1b9cc4a657923c7208d8bbf41d34c707f7a3d

**Comment by livecode-vulcan** [comment]
😎 test success 05c1b9cc4a657923c7208d8bbf41d34c707f7a3d
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/35)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/35)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/35)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/35)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/35)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/35)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/35)

**Pull request merged**